nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/model/GlobalApprovalStatus.java

```diff
@@ -0,0 +1,19 @@
+package no.sikt.nva.nvi.common.service.model;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum GlobalApprovalStatus {
+
+    APPROVED("Approved"), PENDING("Pending"), REJECTED("Rejected"), DISPUTE("Dispute");
```

Comment:
I suspect that it is "In Dispute" or to keep the pattern "Disputed", but it is likely not important